bin/runtime/src/lib.rs

```diff
@@ -118,6 +119,9 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     impl_version: 1,
     apis: RUNTIME_API_VERSIONS,
     transaction_version: 1,
+    /// Version of the state implementation used by this runtime.
+    /// Use of an incorrect version is consensus breaking.    
+    state_version: 1,
```

Comment:
We can spawn a task for it, but I guess update procedure to testnet/mainnet needs to be careful thought.